From 135d7acdb2c9e6c18ec14f174ea1a84691975df3 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 25 Aug 2017 10:39:04 -0700
Subject: [PATCH] MANTA-3396 muskie's "mlocate" alias uses incorrect bash alias
 syntax Reviewed by: Tim Kordas <tim.kordas@joyent.com> Approved by: Kody A
 Kantor <kody.kantor@gmail.com>

---
 boot/setup.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/boot/setup.sh b/boot/setup.sh
index fd34010..98dcc6e 100644
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -7,7 +7,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 set -o xtrace
@@ -209,7 +209,7 @@ manta_setup_muskie_rsyslogd
 
 manta_common_setup_end
 
-#Setup the mlocate alias
-echo "alias=mlocate'/opt/smartdc/muskie/bin/mlocate -f /opt/smartdc/muskie/etc/config.json'" >> $PROFILE
+# Setup the mlocate alias
+echo "alias mlocate='/opt/smartdc/muskie/bin/mlocate -f /opt/smartdc/muskie/etc/config.json'" >> $PROFILE
 
 exit 0
-- 
2.21.0

